app/controllers/gobierto_admin/gobierto_core/templates_controller.rb

```diff
@@ -5,8 +5,8 @@ module GobiertoCore
     class TemplatesController < BaseController
       def index
         template = ::GobiertoCore::Template.first
-        if current_site_has_custom_template?(template.template_path)
-          @site_template = current_site_custom_template(template.template_path).first
+        if ::GobiertoCore::SiteTemplate.current_site_has_custom_template?(current_site, template.template_path)
```

Comment:
This is duplicated in `index` and `edit` actions. Could we extract it to a method or filter and remove the duplication?